content/general/concepts/what-is-UUID/what-is-UUID.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'What is UUID?'
+Description: 'UUID is a universally unique identifier. It can be generated on any computer without the use of a centralized authority. It's a string of 128 bits that is supposed to be unique with little to no chance of clashing with any other UUID produced by anybody else in the world. UUIDs are usually used in database tables as unique keys. They are useful any time you need an ID that needs to be unique in an application or system.'
+Subjects:
+	- 'Computer Science'
```

Comment:
Not sure if the tabs on lines 5 and 8 were intentional, but let's bring them back in line with the other bullet points (2 spaces from the beginning of the line).